### PR TITLE
Feature/ted4 10

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: ted_sws/data_manager/services/daily_notices_metadata_services.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+from datetime import date, datetime, timedelta
```
Comment (CaptainOfHacks) [inline ted_sws/data_manager/services/daily_notices_metadata_services.py:1]: Migrate this file into supra_notice services folder and tests too

File: ted_sws/core/model/supra_notice.py [hunk 2]
```diff
@@ -49,3 +52,50 @@ class DailySupraNotice(SupraNotice):
...
+
+
+class DailyNoticesMetadataABC(PropertyBaseModel):
+    class Config:
+        underscore_attrs_are_private = True
+        validate_assignment = True
+        orm_mode = True
+
+
+NOTICE_STATUSES_DEFAULT_STATS = {str(notice_status): 0 for notice_status in NoticeStatus}
```
Comment (CaptainOfHacks) [inline ted_sws/core/model/supra_notice.py:64]: Constants should be after imports

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
Revise comments

File: ted_sws/data_manager/services/daily_notices_metadata_services.py (hunk 1 above)
Comment (duprijil) [reply ted_sws/data_manager/services/daily_notices_metadata_services.py]: done

File: ted_sws/core/model/supra_notice.py (hunk 2 above)
Comment (duprijil) [reply ted_sws/core/model/supra_notice.py]: done

File: ted_sws/core/model/supra_notice.py [hunk 3]
```diff
@@ -49,3 +52,50 @@ class DailySupraNotice(SupraNotice):
     ted_publication_date: date
     validation_report: Optional[SupraNoticeValidationReport] = None
     validation_summary: Optional[ValidationSummaryReport] = None
+
+
+class DailyNoticesMetadataABC(PropertyBaseModel):
```
[the remark this answers is not in the document]
Comment (duprijil) [reply ted_sws/core/model/supra_notice.py]: done

**Pull request merged**